Options.py

```diff
@@ -860,6 +860,8 @@ class ItemDict(OptionDict):
     verify_item_name = True
 
     def __init__(self, value: typing.Dict[str, int]):
+        if any(item_count is None for item_count in value.values()):
+            raise Exception("Item count option is a dictionary, not a set. Please provide positive integer values.")
```

Comment:
```suggestion
        if any(item_count is None for item_count in value.values()):
            raise Exception("Items must have counts associated with them. Please provide positive integer values in the format \"item\": count .")
```
Your average user isn't going to know what a dictionary vs a set is. Instead, it would be better to state that it needs a quantity and explain how it should be formatted. If you do not like how I worded it, anything similar would be better.

Reply:
Thanks! Could not come up with anything better.